common/utils/shuffle.ts

```diff
@@ -0,0 +1,17 @@
+export default function shuffle(array: any[]) {
+  const finalArray = [...array];
+
+  if (finalArray.length === 0 || finalArray.length === 1) {
+    return finalArray;
+  }
+
+  while (finalArray.toString() === array.toString()) {
+    // Fisher-Yates shuffle
+    for (let i = finalArray.length - 1; i > 0; i--) {
+      const j = Math.floor(Math.random() * (i + 1));
+      [finalArray[i], finalArray[j]] = [finalArray[j], finalArray[i]];
+    }
+  }
+
+  return finalArray;
+}
```

Comment:
We already use lodash in a few places, so might just be easier to `import shuffle from 'lodash/shuffle';`

https://lodash.com/docs#shuffle

Reply:
@wbobeirne 
Excellent, I removed shuffle and replaced with `lodash`.